### ISSUE TITLE
Documenting the code

### ISSUE THREAD

**Issue by dorukayhan** [team1389/Team1389-Robot-Code-Base#2]
Can we write comments in the code to clearly describe what does each class/method do after the world champs?

**Comment by jeprinz** [comment]
Yes good idea we should do this. Also feel free to ask me questions in real life.
